src/EnergyPlus/FileSystem.cc

```diff
@@ -240,7 +240,12 @@ namespace FileSystem {
 
     void moveFile(std::string const &filePath, std::string const &destination)
     {
+#ifdef _WIN32
+        //Note: on Windows, rename function doesn't always replace the existing file so MoveFileExA is used
+        MoveFileExA(filePath.c_str(), destination.c_str(), MOVEFILE_COPY_ALLOWED | MOVEFILE_REPLACE_EXISTING);
```

Comment:
This seems totally fine.  I'm relying on CI a little as I try to avoid building on Windows.  CI is *super* happy with this branch, so I think this is good to go.  The only caveat I can find about this function is that it is missing in the Windows Phone :tm: linkages, so if someone is planning on building E+ for embedding in their Windows phone app, we'd need to have a fallback. :cold_sweat:

Reply:
@Myoldmopar I'll test here.

Reply:
Thanks!